app/src/pages/tracing/TracesTable.tsx

```diff
@@ -104,7 +104,7 @@ export function TracesTable(props: TracesTableProps) {
                 statusCode
                 startTime
                 latencyMs
-                tokenCountTotal
+                tokenCountTotal: cumulativeTokenCountTotal
```

Comment:
might not want to alias this this way - I'd just change it. This is probably making the sorting confused?